samples/snippets/src/main/java/com/example/bigquerystorage/WriteCommittedStream.java

```diff
@@ -64,7 +65,9 @@ public static void writeCommittedStream(String projectId, String datasetName, St
       try (JsonStreamWriter writer =
           JsonStreamWriter.newBuilder(writeStream.getName(), writeStream.getTableSchema())
               .build()) {
-        // Write two batches to the stream, each with 10 JSON records.
+        // Write two batches to the stream, each with 10 JSON records. A writer should be
+        // used for as much writes as possible. Create a writer for just one write is an
```

Comment:
Create > Creating

Reply:
Done. and added some more change to buffered stream. PTAL.